lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -60,15 +66,91 @@ def diode_params_from_voltage(
     )
 
 
-def viscosity_of_water(temperature):
-    """Computes the viscosity of water in [Pa*s] at a particular temperature.
+def density_of_water(temperature, molality, pressure=0.101325):
+    """Determine the density of water with NaCl.
+
+    This model is based on [1]_.
+
+    Parameters
+    ----------
+    temperature : array_like
+        Temperature in C
+    molality : float
+        Molality NaCl [mol/kg]
+    pressure : float, optional
+        Pressure (default: 0.101325) [MPa].
+
+    Raises
+    ------
+    ValueError
+        When the provided values are outside the valid range of this model. The valid ranges are:
+        Temperature (20°C <= T < 150°C), pressure <= 35 MPa and molality <= 6 mol/kg.
+
+    References
+    ----------
+    .. [1] Kestin, J., Khalifa, H. E., & Correia, R. J. (1981). Tables of the dynamic and
+       kinematic viscosity of aqueous NaCl solutions in the temperature range 20–150 C and
+       the pressure range 0.1–35 MPa. Journal of physical and chemical reference data, 10(1),
+       71-88.
+    """
+    import scipy.constants
+
+    _check_salt_model_validity("Density", temperature, pressure, molality)
+
+    temperature_k = scipy.constants.convert_temperature(temperature, "C", "K")
+    weight = 58.4428 / 1000  # kg/mol for NaCl
+    mass_fraction = (
+        molality * weight / (1.0 + molality * weight)
+    )  # mol NaCl/kg water -> kg NaCl/kg total
+
+    i_ab = np.arange(-2, 3, 1)
+    i_other = np.arange(0, 3, 1)
```

Comment:
I'll just call them `powers_ab` and `powers_other`. The former are powers for the a and b polynomials while the other one is shared between all other polynomials. The reason I had called them `i` is because that is what they were called in the paper. See page 73, equation 12, 13, 14.

Reply:
yeah I realized going through the rest of the internal api that there's probably just no good names for most of these variables. this is fine though